packages/ui/src/forum/hooks/useForumCategories.ts

```diff
@@ -1,29 +1,39 @@
-import { useGetForumCategoriesQuery } from '@/forum/queries'
-import { asForumCategory, CategoryStatusType } from '@/forum/types'
+import { useGetForumCategoriesQuery, useGetArchivedForumCategoriesQuery } from '@/forum/queries'
+import { asForumCategory, asArchivedForumCategory, CategoryStatusType } from '@/forum/types'
 
 interface Props {
   isRoot?: boolean
-  isArchive?: boolean
 }
 
 export const ActiveStatus: CategoryStatusType = 'CategoryStatusActive'
 export const ArchivedStatus: CategoryStatusType = 'CategoryStatusArchived'
 
-export const useForumCategories = ({ isRoot, isArchive }: Props = {}) => {
+export const useForumCategories = ({ isRoot }: Props = {}) => {
   const { loading, data } = useGetForumCategoriesQuery({
     variables: {
       where: {
         ...(isRoot ? { parent: { id_eq: null } } : {}),
-        status_json: { isTypeOf_eq: isArchive ? ArchivedStatus : ActiveStatus },
-        ...(isArchive && !isRoot
-          ? { OR: [{ parent: { id_eq: null } }, { parent: { status_json: { isTypeOf_eq: ActiveStatus } } }] }
-          : {}),
+        status_json: { isTypeOf_eq: ActiveStatus },
       },
     },
   })
-
   return {
     isLoading: loading,
     forumCategories: data?.forumCategories.map(asForumCategory),
   }
 }
+
+export const useArchivedForumCategories = () => {
```

Comment:
We tend to have one main function/component per file so this might be extracted to `useArchivedForumCategories.ts` together with `ArchiveStatus`.